app/app.go

```diff
@@ -588,6 +586,45 @@ func NewBabylonApp(
 	return app
 }
 
+// RegisterServicesWithoutStaking calls the module manager
+// registration services without the staking module.
+func (app *BabylonApp) RegisterServicesWithoutStaking() {
+	// removes the staking module from the register services
+	stkModTemp := app.ModuleManager.Modules[stakingtypes.ModuleName]
+	delete(app.ModuleManager.Modules, stakingtypes.ModuleName)
+
+	if err := app.ModuleManager.RegisterServices(app.configurator); err != nil {
+		panic(err)
+	}
+
+	// adds the staking module back it back
+	app.ModuleManager.Modules[stakingtypes.ModuleName] = stkModTemp
+}
+
+// RegisterStakingQueryAndMigrations registrates in the configurator
+// the x/staking query server and its migrations
+func (app *BabylonApp) RegisterStakingQueryAndMigrations() {
```

Comment:
We still need to register the staking migrations, so every new migration in https://github.com/cosmos/cosmos-sdk/blob/32969cf7cc602bb380f78dee23f0f0a747728791/x/staking/module.go#L141 we need to apply here as well